src_erl/Communication_Layer/http_Nerlserver/input/jsonArch1PC.json

```diff
@@ -28,12 +28,12 @@
     {
       "name": "w1",
       "id": 0,
-      "args": "[561,20,15,13,10,6]@0.001@[0,2,3,3,3,0]@6@0@561@6"
+      "args": "[3,2,1]@0.001@[0,2,0]@6@0@3@1"

```

Comment:
Didn't we move to explicit parameters description? 
@6@3@1 means nothing to new user

Reply:
Yes. it's an old commit